src/dotnet/APIView/APIViewWeb/Repositories/NotificationManager.cs

```diff
@@ -29,6 +31,8 @@ public class NotificationManager
         private const string REPLY_TO_HEADER = "In-Reply-To";
         private const string REFERENCES_HEADER = "References";
 
+        static TelemetryClient _telemetryClient = new(TelemetryConfiguration.CreateDefault());
```

Comment:
Does dependency injection not work for TelemetryClient? Since we are already adding the AppInsights services in Startup, I would have expected it to be possible to inject the client in the constructor.

Reply:
Let me check it before I merge this PR.